doc/contributing/using-a-custom-version-of-cesium.md

```diff
@@ -2,22 +2,41 @@
 
 What if you need to make changes to [Cesium](https://github.com/AnalyticalGraphicsInc/cesium) while working on TerriaJS?
 
+TerriaJS is using a fork of Cesium monorepo located at https://github.com/terriajs/cesium, and similar to the upstream repo it consists of two packages:
+
+- [terriajs-cesium](https://www.npmjs.com/package/terriajs-cesium) corresponds to the upstream [@cesium/engine](https://www.npmjs.com/package/@cesium/engine/)
+- [terriajs-cesium-widgets](https://www.npmjs.com/package/terriajs-cesium-widgets) corresponds to the upstream [@cesium/widgets](https://www.npmjs.com/package/@cesium/widgets/)
+
 The process of using a custom version of Cesium is much the same as using a custom version of TerriaJS. See the [Development Environment](development-environment.md#building-a-terriamap-against-a-modified-terriajs) for information on setting up and using `yarn`. To clone Cesium, do:
 
-```
+```sh
 cd packages
-git clone -b terriajs https://github.com/TerriaJS/cesium.git
-cd ..
+git clone https://github.com/TerriaJS/cesium.git terriajs-cesium
 ```
 
-It is important that you use the `terriajs` branch of [TerriaJS/cesium](https://github.com/TerriaJS/cesium) because it contains important changes to Cesium that are necessary for it to work with TerriaJS. If you need to use a different branch of Cesium, you will need to merge that branch with the changes in the `terriajs` branch.
+Make sure that the `terriamap/package.json` workspaces list contains the following packages:
+
+- "packages/terriajs-cesium/packages/engine",
+- "packages/terriajs-cesium/packages/widgets"
+
+Check the versions of `terriajs-cesium` and `terriajs-cesium-widgets` in `terriamap/package.json` and `terriamap/packages/terriajs/package.json` and make sure they are the same so yarn can properly create symlinks.
 
-And then run:
+Then run:
 
 ```
 yarn install
 ```
```

Comment:
@zoran995  does running `yarn install` before building the `cesium` repo below give an error for you, because post-install terria tries to copy the cesium workers and other assets which it cannot find? 

I think before `yarn install` from terriamap, we need to do:
```bash
cd packages/terriajs-cesium
npm install
npm run release
```

and then:
```bash
yarn install # from terriamap
```

![image](https://github.com/user-attachments/assets/a4638d81-93b5-4972-beb5-1d5a3ec6c7c8)

Either that, or fix `post-npm-install` step in terriajs/gulpfile to not fail when it cannot find the cesium assets.

Reply:
added a section for installing cesium dependencies and building the package